enderio-base/src/main/java/com/enderio/base/common/capability/EntityFilterCapability.java

```diff
@@ -125,7 +125,7 @@ public boolean test(EntityType<?> entity) {
     public record Component(List<StoredEntityData> entities, boolean nbt, boolean invert) {
         public static Codec<Component> CODEC = RecordCodecBuilder.create(componentInstance -> componentInstance
```

Comment:
We can't change the codec like this as it will break existing saves. If you look at ItemFilterCapability you'll see LEGACY_CODEC and NEW_CODEC. If you could follow that pattern for updating the Codec format instead that'd be fantastic. Thanks!

Reply:
Ahh I see, didn't think about that. Fixing it right away!

Reply:
Done and tested :)